.github/workflows/update-providers-ci-mgmt.yml

```diff
@@ -1,4 +1,50 @@
 name: Update Providers with new bridge version via ci-mgmt
 on:
+  release:
+    types: [published]
```

Comment:
Hmm. I publish releases for pf/* tags separately from normal release. Will this confuse it?

Reply:
Good catch, yes it's getting confused.